src/lib/viewers/BaseViewer.js

```diff
@@ -988,6 +988,9 @@ class BaseViewer extends EventEmitter {
         if (this.areNewAnnotationsEnabled() && this.annotationControls) {
             this.annotator.addListener('annotations_create', this.handleAnnotationCreateEvent);
         }
+
+        // Signal that the annotator and listeners have been loaded.
+        this.emit('annotations_loaded');
```

Comment:
Should this be `annotator_loaded` to follow the existing event's convention somewhat. Would `annotator_ready` work, since the Annotations SDK is loaded much earlier?

Reply:
Yeah, that is much better!